cmd/zoekt-mirror-gerrit/main.go

```diff
@@ -147,7 +148,7 @@ func main() {
 			continue
 		}
 
-		cloneURL, err := url.Parse(strings.Replace(projectURL, "${project}", k, -1))
+		cloneURL, err := url.Parse(strings.Replace(projectURL, "${project}", k, 1))
```

Comment:
Why does this need to change?

Reply:
It does not really need to be changed. It was just because only one occurrence of "${project}" is expected to be replaced.